packages/story-editor/src/app/story/useStoryReducer/test/addElement.js

```diff
@@ -75,4 +80,93 @@ describe('addElement', () => {
     });
     expect(result.selection).toStrictEqual(['321']);
   });
+
+  it('should add an element with a unique product id', () => {
```

Comment:
See https://github.com/GoogleForCreators/web-stories-wp/pull/11727 for more tests.